aws-ecs/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Telegraf agent configured for Wavefront output intended to be used as it's own deployment to collect metrics
+
+FROM telegraf:1.17.2
```

Comment:
Let's not depend on a specific version, make it generic. Can you test this?
```
FROM telegraf:latest
```

Reply:
Changed